content/docs/latest/spire-about/comparisons.md

```diff
@@ -16,7 +16,7 @@ Secrets managers typically control, audit and securely store sensitive informati
 
 A common architectural challenge in deploying secrets managers is how to securely store the credential that is used by the workload to access the secret store itself. This is sometimes called "credential zero", the "bootstrap credential", or more broadly, the process of "secure introduction".
 
-By contrast, while SPIRE does _generate_ SPIFFE identities that can be used to [authenticate to other systems](/docs/latest/spire-integrations/use-cases/), SPIRE does not aim to store existing keys (such as a database password) on behalf of a workload.
+By contrast, while SPIRE does _generate_ SPIFFE identities that can be used to [authenticate to other systems](/docs/latest/spire-about/use-cases/), SPIRE does not aim to store existing keys (such as a database password) on behalf of a workload.
```

Comment:
I think this page was just moved in #194, and the alias went one level too deep for this link to work (`/docs/latest/spire-integrations/use-cases/use-cases` instead of `/docs/latest/spire-integrations/use-cases`.

Reply:
nice catch!